### PR TITLE
perf(ui): update loader

### LINKED ISSUE #4808: Pipeline run view: CPU, lag
Pipeline run page => High CPU Usage, lag etc..

### PR DESCRIPTION
**Pull request by yesnault**
Signed-off-by: Yvonnick Esnault <yvonnick.esnault@corp.ovh.com>

GPU: 30% before, 2% after 
CPU: 23% before, 6% after.

close #4808

### CODE REVIEW

**Comment by yesnault** [comment]
with 

```html
 <div *ngIf="loader" class="sync alternate blue icon">
                <span class="">{{value}}</span>
            </div>
```


Capture d’écran 2020-02-29 à 16 07 16


with 
```
 <i *ngIf="loader" class="spinner blue icon">
                <span class="">{{value}}</span>
            </i>
```
Capture d’écran 2020-02-29 à 16 06 41

--> results:
GPU: 30% before, 2% after 
CPU: 23% before, 6% after.

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#12593.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✘

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#12595.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Comment by yesnault** [comment]
Results, with no data in network (Task View) and the lowest numbers.

With CSS Animation:
- Workflow view GPU: 13% CPU: 11%
- Pipeline view GPU 20% CPU: 12%

Gif:
- Workflow view GPU: 4.3% CPU: 6%
- Pipeline view GPU 6.6% CPU: 8%

Static:
- Workflow view GPU: 1.9% CPU: 4.9%
- Pipeline view GPU 1.9% CPU: 6%

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#12601.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✘

File: ui/src/app/shared/status/status.icon.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <div class="status-icon" [ngSwitch]="status">
     <div class="wrapper">
         <ng-container *ngSwitchCase="pipelineStatusEnum.BUILDING">
-            <div *ngIf="loader" class="ui small active inline blue loader">
-                <span class="">{{value}}</span>
-            </div>
+            <i *ngIf="loader">
+                <img id="gifloader" src="/assets/images/loader.gif">
```
Comment (sguiheux) [inline ui/src/app/shared/status/status.icon.html]: what about <span class="">{{value}}</span> ?
If not use anymore, clean entire component

**Comment by yesnault** [comment]
Screenshots with gif (dark and not)

Capture d’écran 2020-03-02 à 17 45 45
Capture d’écran 2020-03-02 à 17 45 52

**Pull request merged**